lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -29,16 +32,6 @@ const propertiesWithShorthandNotation = new Set([
 	'inset',
 ]);
```

Comment:
Can you add the alias of `grid-gap`?
i.e. `gap`

Reply:
Out of scope of the PR. Please add it in another PR?